databox/client/lib/Databox.ts

```diff
@@ -15,9 +15,20 @@ export default class Databox<
 > {
   #databoxInternal: DataboxInternal<TTable, TFunction, TComponents>;
 
+  public get paymentAddress(): string | undefined {
+    return this.#databoxInternal.components.paymentAddress;
+  }
+
+  public get giftCardIssuerIdentity(): string | undefined {
+    return this.#databoxInternal.components.giftCardIssuerIdentity;
+  }
+
```

Comment:
I prefer when all the getters and setters are together. This file has some above the constructor and some below it.

Reply:
Yeah, will fix. Not on purpose